editor/src/clj/editor/build.clj

```diff
@@ -42,33 +40,20 @@
 
 (defn build-project!
   [project node evaluation-context extra-build-targets old-artifact-map render-progress!]
-  (if-let [extension-error (extensions/execute-hook! project :on-build-started {:exception-policy :as-error
```

Comment:
Mostly indentation because we moved the editor script hooks into the `async-build!` function as separate phases.